config-staging.json

```diff
@@ -63,7 +63,10 @@
         "https://api.blockchain.info/bnb/nodes/rpc"
```

Comment:
Are you sure this is working? Who uses this? Isn't this a proxy to our internal node?